Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -461,6 +444,8 @@ namespace OpenInfraPlatform {
 					// |	 |	|
 					// 2-----3	---> x
 
+					std::shared_ptr<carve::input::PolylineSetData> polylineData = std::shared_ptr<carve::input::PolylineSetData>();

```

Comment:
```suggestion
					std::shared_ptr<carve::input::PolylineSetData> polylineData = std::make_shared<carve::input::PolylineSetData>();
```
This is how you initialize a new object of type `shared_ptr`. Apply everywhere where suitable.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/4cf7d79ec9d69c7fc074dff32fcc56747ccfbb78